app/src/main/java/edu/wpi/first/shuffleboard/app/components/WidgetPane.java

```diff
@@ -100,6 +100,7 @@ public WidgetPane() {
       if (highlight) {
         getChildren().add(gridHighlight);
         gridHighlight.toFront();
+        gridHighlight.setMouseTransparent(true);
```

Comment:
fixes drag-and-drop regression from #259

Reply:
So _that's_ what that was! It's been driving me nuts.